release_docs/RELEASE.txt

```diff
@@ -1172,6 +1172,15 @@ Bug Fixes since HDF5-1.12.0 release
 
     Configuration
     -------------
+    - Corrected path searched by CMake find_package command
+
+      The install path for cmake find_package files had been changed to use
+        "share/cmake"
```

Comment:
@byrnHDF this causes a regression on at least Unix.
The supported directory names are:
https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure

Reply:
For your tests, avoid setting `HDF5_DIR` to the CMake sub-path. Instead, just hint [CMAKE_PREFIX_PATH](https://cmake.org/cmake/help/latest/envvar/CMAKE_PREFIX_PATH.html) to the root/prefix of the whole HDF5 installation. That's all that should be needed to hint a HDF5 install if the supported directory naming scheme is used :)